main.py

```diff
@@ -141,8 +142,12 @@ def load_user(userid):
     login_manager.anonymous_user = load_anonymous_user
 
     stripe.api_key = app.config["STRIPE_SECRET_KEY"]
-    pywisetransfer.environment = app.config["TRANSFERWISE_ENVIRONMENT"]
-    pywisetransfer.api_key = app.config["TRANSFERWISE_API_TOKEN"]
+    global wise
```

Comment:
Not at all a blocker, but this feels like a sign that the interface for `pywisetransfer` might not be great.  Use of globals feels non-ideal and if there's a way to avoid that (again) in future that'd be nice.

Reply:
Using singleton globals here is a Flask pattern, although they also usually have a separate `init_app` that actually configures the modules. I'm unsure the answer is to go back to Stripe's pattern of storing state in the module, but perhaps it's worth adding a Flask wrapper specifically to support this pattern?

Reply:
Yep, some kind of flask plugin is probably the way to go, middle-term.

That said: I think I've also seen some kind of app-level global storage on the Flask `app` object before?  (`app.g.<your-variable-name> = <value>`, or similar - https://flask.palletsprojects.com/en/2.0.x/api/#flask.g?)

Shorter-term: fine as-is, albeit maybe with a comment to indicate that it might not be ideal practice (unless we think it's fine, in which case don't bother).